modules/get-started/pages/release-notes/helm-charts.adoc

```diff
@@ -12,11 +12,79 @@ See also:
 * xref:upgrade:k-compatibility.adoc[]
 * xref:upgrade:k-rolling-upgrade.adoc[]
 
-== Redpanda chart v5.9.21
+== Redpanda chart v25.1-k8s-beta1
+
+link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.1.x/charts/redpanda/CHANGELOG.md[Changelog].
+
+See also: xref:deploy:deployment-option/self-hosted/kubernetes/k-25.1-beta.adoc[]
+
+=== Redpanda Console v3
+
+This beta version deploys Redpanda Console v3, which includes unified authentication and authorization between the Console and Redpanda, including user impersonation. For more information, see xref:console:config/security/authentication.adoc[].
+
+This version does not support Redpanda Console v2.
+
+=== Schema validation
+
+The Redpanda Helm chart now includes built-in schema validation to ensure configurations conform to expected standards. With schema validation enabled, you must ensure your overrides include valid fields.
```

Comment:
We've actually always had schema validation, we've just made it more strict. So given the values:

```yaml
statefulset:
  replicas: "Not a number"
  liables:
    there-is: a-typo-in-this-field
```

In v5.9.x, you'd get an error about `statefulset.replicas` needing to be a number but no error about `liables`

In v25.1.x, you'll get an error about `statefulset.replicas` needing to be a number AND an error that `liables` isn't an allowed property.